File: Retos/Reto #0 - EL FAMOSO FIZZ BUZZ [Fácil]/dart/bushi.dart
void main() {
  for (var i = 1; i <= 100; i++) {
    var result = "$i. ";

    if (i % 3 == 0) {
      result += "fizz";
    }
    if (i % 5 == 0) {
      result += "buzz";
    }

    print("$result\n");
  }
}

File: Retos/Reto #0 - EL FAMOSO FIZZ BUZZ [Fácil]/kotlin/bushi.kt
fun main() {
    for (i in 1..100) {
        var result = "$i. "

        if (i % 3 == 0) {
            result += "fizz"
        }
        if (i % 5 == 0) {
            result += "buzz"
        }

        println(result)
    }
}